samples/word/50-document/manage-custom-xml-part.yaml

```diff
@@ -0,0 +1,222 @@
+order: 7
+id: word-document-manage-custom-xml-part
+name: Manage a CustomXmlPart
+description: 'This sample shows how to add, query, edit, and delete a custom XML part in a document.'
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#add-custom-xml-part").click(() => tryCatch(addCustomXmlPart));
+        $("#query").click(() => tryCatch(query));
+        $("#insert-attribute").click(() => tryCatch(insertAttribute));
+        $("#insert-element").click(() => tryCatch(insertElement));
+        $("#delete-custom-xml-part").click(() => tryCatch(deleteCustomXmlPart));
+
+        async function addCustomXmlPart() {
+          // Add a custom XML part.
+          await Word.run(async (context) => {
+            // You must have the xmlns attribute to populate the CustomXmlPart.namespaceUri property.
+            const originalXml =
+              "<Reviewers><Reviewer>Juan</Reviewer><Reviewer>Hong</Reviewer><Reviewer>Sally</Reviewer></Reviewers>";
+            const customXmlPart = context.document.customXmlParts.add(originalXml);
+            customXmlPart.load("id");
+            const xmlBlob = customXmlPart.getXml();
+
+            await context.sync();
+
+            const readableXml = addLineBreaksToXML(xmlBlob.value);
+            console.log("Added custom XML part:");
+            console.log(readableXml);
+
+            // Store the XML part's ID in a setting so the ID is available to other functions.
+            const settings = context.document.settings;
+            settings.add("ContosoReviewXmlPartId", customXmlPart.id);
+
+            await context.sync();
+          });
+        }
+
+        async function query() {
+          // Query the specified custom XML part.
```

Comment:
Since there isn't a parameter passed as the "specified" part, consider explicitly calling out what you're searching for in the comment. Like, "Query a custom XML part with an established key for elements matching "/Reviewers/Reviewer"".

Reply:
I think it's okay to be specific. It helps users tie the code to actions. We do this all the time in Scripts.